Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -941,9 +982,21 @@ private void UpdateRigVisibilityInInspector()
         }
 
         [Header("Events")]
+        /// <summary>
+        /// Event that gets fired when interaction with a rotation handle starts.
+        /// </summary>
         public UnityEvent RotateStarted = new UnityEvent();
+        /// <summary>
+        /// Event that gets fired when interaction with a rotation handle ends.
```

Comment:
```suggestion
        /// Event that gets fired when interaction with a rotation handle stops.
```
Used _stops_ below, so we should probably use _stops_ here to be consistent with the event names.